frontend/www/js/omegaup/components/Countdown.vue

```diff
@@ -1,21 +1,50 @@
 <template>
-  <div class="base-timer">
-    <span class="base-timer__label">
+  <div>
+    <span>
       {{ formattedTimeLeft }}
     </span>
   </div>
 </template>
 
 <script lang="ts">
-import { Vue, Component, Prop } from 'vue-property-decorator';
+import { Vue, Component, Prop, Watch } from 'vue-property-decorator';
 import * as time from '../time';
 
 @Component
 export default class Countdown extends Vue {
-  @Prop() timeLeft!: number;
+  @Prop() startTime!: Date;
+
+  timePassed = 0;
+  timerInterval = 0;
+
+  get timeLeft() {
+    if (!this.startTime) {
+      return 0;
+    }
+    const timeLimit = this.startTime.getTime() - Date.now();
+    return timeLimit - this.timePassed;
```

Comment:
por qué es necesario el `timePassed`? `Date.now()` ya debería incorporar el tiempo que ya pasó. onda

```suggestion
    return this.startTime.getTime() - this.currentTime;
```

el `this.currentTime` es para que se pueda establecer en L43 (mediante `this.timerInterval = window.setInterval(() => {this.currentTime = Date.now();}, 1000);`) y que Vue detecte que _algo_ cambió y se reevalúe esta propiedad.

Reply:
Lo de `timePassed` era también por el ejemplo en el que me basé. pero si se puede quitar sin problemas.